source/2019-09-20-the-ember-times-issue-116.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: The Ember Times - Issue No. 116
-author: the crowd
+author: Chris Ng, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/09/20-the-ember-times-issue-116.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
-
+Check out the new Octane Super Rentals Tutorial 🚀,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Octane Super Rentals Tutorial 🚀](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@chancancode](https://github.com/chancancode) and [@vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing content and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/) and check out the tutorial.
```

Comment:
There is a good chance that https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/ would have been filled in by the time we send this. If it's up, it would be great to change the link to the introduction chapter instead!

Reply:
@chancancode ok, keep us posted. Are you saying that this link https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/ might be updated? I'm eyeing a post-4:30 PM pacific release

Reply:
Yes! Working on it now!

Reply:
Cool will update the links in the blog